tests/unit/ServiceProviderTest.php

```diff
@@ -19,6 +19,24 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('kinesis.aws.key', 'key');
         $app['config']->set('kinesis.aws.secret', 'secret');
         $app['config']->set('kinesis.level', 1);
+
+
+        $app['config']->set('logging.channels', [
+            'kinesis' => [
+                'driver' => 'monolog',
+                'handler' => \PodPoint\KinesisLogger\Monolog\KinesisHandler::class,
+                'with' => [
+                    'streamName' => config('kinesis.stream'),
+                    'level' => config('kinesis.level'),
+                ],
+                'formatter' => \PodPoint\KinesisLogger\Monolog\KinesisFormatter::class,
+                'formatter_with' => [
+                    'name' => config('app.name'),
+                ],
+            ],
+        ]);
```

Comment:
Can we rewrite these to be reflective of the new channel method of configuring logs. Ideally we should be testing this with all major support Laravel versions. See https://github.com/Pod-Point/countries/blob/master/.github/workflows/run-tests.yml on how to do this with Actions and the matrix strategy.
Also, can we add more meaningful tests? As a minimum we should be able to mock the Kinesis client, setup the channel, send a log, and assert the client gets `putRecord` called with the expected body. We should do this for Laravel 6, 7 & 8 with both versions of PHP.